backstage/values.yaml

```diff
@@ -119,6 +119,7 @@ postgresql:
   initdbScriptsSecret: backstage-postgresql-initdb
 
 appConfig:
+  tenant: roadie
```

Comment:
This line has to be moved to line 120 for example so that the tenant value becomes a root level value

Reply:
:+1: moved this up one level.